lib/dirac.js

```diff
@@ -471,6 +471,10 @@ dirac.tx = {
       tx[ tbl ] = new Dal( client );
       tx[ tbl ].befores = dirac.dals[ tbl ].befores;
       tx[ tbl ].afters  = dirac.dals[ tbl ].afters;
+      tx[ tbl ].lock = function(mode, callback) {
```

Comment:
It's definitely a weird setup though. Regular dal calls just grab a random client, and supporting transactions calls for reusing the same client. 

Maybe there's a better way to pattern transactions